example/test_problem/Hydro/AGORA_IsolatedGalaxy/README

```diff
@@ -34,3 +34,17 @@ Note:
 3. Run the script "download_ic.sh" to download the low-resolution initial condition files for this test
    --> It's the same script used in the "AgoraGalaxy" test problem of Enzo
 4. Some handy yt analysis scripts are put at "yt_script"
+
+
+High-resolution setup and notes:
+========================================
+To reproduce the AGORA high-resolution run presented in the GAMER-2 paper (Schive et al. 2018), follow the above steps with the following modifications:
+1. Download the high-resolution initial conditions by executing "sh download_ic_high_res.sh"
+2. Adjust the following parameters in "Input__Parameter":
+    PAR_NPAR                      21250000    # from the default value of 212500
+    MAX_LEVEL                     9           # from the default value of 7
+
+
```

Comment:
Dear @BarryTC (and @hyschive),

I have recently tried to reproduce the results of the AGORA isolated galaxy in the [GAMER-2 paper](https://doi.org/10.1093/mnras/sty2586).
Thank you for preparing this pull request. It helps me a lot.

Following your setups of high-resolution runs, here is my result of the star formation rate (to be compared to Figure 17. in the GAMER-2 paper):
![image](https://github.com/user-attachments/assets/eb650766-e370-4c16-a3c2-bd8351313dbf)
May I confirm with you whether this is consistent with your results?


When I compared this test problem in GAMER to the GAMER-2 paper, I found that besides the high-resolution IC, `PAR_NPAR` and `MAX_LEVEL`, it seems we also need to adjust the star formation parameters and the refinement criteria.

### Additional Changes
- In Section 4.4.1 of the GAMER-2 paper, the gas density threshold is $n_\mathrm{H,thres} = 20\ \mathrm{cm}^{-3}$
   - In "Input__Parameter",
     the default `SF_CREATE_STAR_MIN_GAS_DENS   5.0e0` should be changed to
     `SF_CREATE_STAR_MIN_GAS_DENS   2.0e1`.
- In Section 4.4.1 of the GAMER-2 paper, the minimum star particle mass is $m_{*} = 2\times 10^{3}\ M_\odot$
   - In "Input__Parameter",
     the default `SF_CREATE_STAR_MIN_STAR_MASS  3.2e4` should be changed to
     `SF_CREATE_STAR_MIN_STAR_MASS  2.0e3`.
- In Table 3. of the GAMER-2 paper, the maximum particle mass in a cell is $1\times 10^6\ M_\odot$.
   - In "Input__Flag_ParMassCell",
     the default `1.0035496e-1` should be changed to
     `1.0035496e-3`.
- In Table 3. of the GAMER-2 paper, the maximum gas mass in a cell is $3.4\times 10^{2}\ M_\odot$.
   - In "Input__Flag_Rho",
     the default
     ```
     # Level                    Density
      0           2.000480890274048e-08
      1           1.600384712219238e-07
      2           1.280307769775391e-06
      3           1.024246215820313e-05
      4           8.193969726562500e-05
      5           6.555175781250000e-04
      6           5.244140625000000e-03
      7           4.195312500000000e-02
      8           3.356250000000000e-01
      9           2.685000000000000e-00
     10           2.148000000000000e+01
     ```
     should be changed to
     ```
     # Level                    Density
      0           3.201222978532314e-10
      1           2.560978382825851e-09
      2           2.048782706260681e-08
      3           1.639026165008545e-07
      4           1.311220932006836e-06
      5           1.048976745605469e-05
      6           8.391813964843749e-05
      7           6.713451171875000e-04
      8           5.370760937500000e-03
      9           4.296608750000000e-02
     10           3.437287000000000e-01
     ```
- In Table 3. of the GAMER-2 paper, the refinement resolves Jeans length by at least $64$ cells.
   - In "Input__Flag_Jeans",
     the default `32.0` should be changed to
     `64.0`

### Results
Including the above changes, here is the resulting star formation rate:
![image](https://github.com/user-attachments/assets/9a46c00e-d1a0-40c5-b969-2128c352dc15)
I think it looks better.

Could you help me check whether these changes make sense?
Thanks!


### Note
- There are still some differences remained:
   - In Table 3. of the GAMER-2 paper, it said "Root grid $64^3$, $10$ refinement levels" and "Five additional levels of statically refined regions above the root grid".
   - In GAMER, it actually is root grid $128^{3}$, $9$ refinement levels, and 4 statically refined levels.
- I am using the single-precision GAMER with the double-precision GRACKLE (as recommended), supported in https://github.com/gamer-project/gamer/pull/365

Reply:
A minor difference is found in @hyschive's previous `Input__Flag_Rho`, which corresponds to the maximum gas mass in a cell is $335.625\ M_\odot$ and is $1/64$ of the default criteria:
```
# Level                         Density
      0           3.125751391053200e-10
      1           2.500601112842560e-09
      2           2.000480890274048e-08
      3           1.600384712219238e-07
      4           1.280307769775391e-06
      5           1.024246215820313e-05
      6           8.193969726562500e-05
      7           6.555175781250000e-04
      8           5.244140625000000e-03
```

, while my `Input__Flag_Rho` provided in the previous comment is set to the maximum gas mass in a cell = $343.729\ M_\odot$ ($= 0.1\times$ mass of each particle in the initial conditions of the disk and the bulge).